src/workerd/api/cache.c++

```diff
@@ -76,6 +76,16 @@ jsg::Promise<jsg::Optional<jsg::Ref<Response>>> Cache::match(jsg::Lock& js,
     CompatibilityFlags::Reader flags) {
   // TODO(someday): Implement Cache API in preview.
   auto& context = IoContext::current();
+  auto traceSpan = context.makeTraceSpan("cache_match"_kjc);
+  auto userSpan = context.makeUserTraceSpan("cache_match"_kjc);
+  TraceContext traceContext(kj::mv(traceSpan), kj::mv(userSpan));
+
+  KJ_IF_SOME(o, options) {
+    KJ_IF_SOME(ignoreMethod, o.ignoreMethod) {
+      traceContext.userSpan.setTag("cache.request.ignore_method"_kjc, ignoreMethod);
```

Comment:
Can you use traceContext.setTag instead of traceContext.userSpan.setTag please?
(this comment applies everywhere in this file please 🙏 
I added that API on purpose so that our internal jaeger tracing can also enjoy your work